CMakeLists.txt

```
@@ -140,9 +142,31 @@ if( DEFINED DD4HEP_WITH_GEANT4 OR DEFINED DD4HEP_WITH_GEAR )
   dd4hep_print_cmake_options( OPTIONAL "Obsolete options: only the following are allowed:" ERROR 1 )
 endif()
 
+IF(DD4HEP_USE_EXISTING_DD4HEP)
+
+  MESSAGE(STATUS "|!!> Using pre-existing DD4hep: ${DD4HEP_USE_EXISTING_DD4HEP}")
+  SET(CMAKE_PROJECT_NAME DD4hep_EXTRA)
```

I think ``DD4hepSelected` would be better.
```suggestion
  SET(CMAKE_PROJECT_NAME DD4hepSelected)
```

---

I agree